src/test/java/org/isf/medstockmovtype/test/Tests.java

```diff
@@ -73,26 +73,26 @@ public void testMovementTypeSets() throws Exception {
 	}
 
 	@Test
-	public void testIoGetMovementType() throws Exception {
+	public void testIofindOne() throws Exception {
```

Comment:
should the method name be 'testIoFindOne()`?  Capital `F`.

Reply:
Yep! Thanks!